tests/e2e-openshift/monitoring/03-assert.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: batch/v1
```

Comment:
You don't actually need to wait for the job to be completed, right?

Reply:
Updated the step to keep the generate traces job running for 3 minutes and check if the job is running in assert. The check is required cause in some cluster environments, there might be issues with image pull or any other issue to start the pods.